src/Libraries/Nop.Services/Customers/CustomerService.cs

```diff
@@ -796,16 +710,15 @@ public virtual async Task<int> DeleteGuestCustomersAsync(DateTime? createdFromUt
         /// </summary>
         /// <param name="customer">Customer</param>
         /// <returns>
-        /// A task that represents the asynchronous operation
-        /// The task result contains the customer full name
+        /// The result contains the customer full name
         /// </returns>
-        public virtual async Task<string> GetCustomerFullNameAsync(Customer customer)
+        public virtual string GetCustomerFullName(Customer customer)

```

Comment:
No need to change this method to the sync version. Let's leave async for consistency.

Reply:
Don't forget to revert the summary comments here.